app/views/tasks/_task_summary.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= link_to task.name, task_path(task.id) %>
+<%= link_to "Edit", "/tasks/#{task.id}/edit" %>
+<%= button_to "Complete", "/tasks/#{task.id}/complete", method: :get %>
```

Comment:
These two lines should be using named routes - in this case, it would be `edit_task_path(task.id)` and `completed_path(task.id)`. Those names come from the `as:` argument you supplied in your routes file.